parlai/chat_service/tasks/overworld_demo/worlds.py

```diff
@@ -252,6 +252,7 @@ class MessengerOverworld(World):
             MessengerOnboardDataTaskWorld,
         ),
         'chat': (MessengerChatOnboardWorld, MessengerChatTaskWorld),
+        'EXIT': (None, None),
```

Comment:
this means that we'll always need `EXIT` as an option? or is this just specific to this setup

Reply:
It's sorta up in the air as to if a service wants to implement an explicit exit. People can always just disconnect